client/components/dashboard/dashboard-directive.html

```diff
@@ -65,7 +74,7 @@
            class="pk-container-footer"
            ng-class="{'pk-section-fixed-height': container.model.container.footer_height}"
            ng-style="{'height': container.model.container.footer_height + 'px'}"
-           markdown-to-html="container.model.container.footer_text | stripHtml">
+           markdown-to-html="container.model.container.footer_text">
```

Comment:
Why are you removing `| stripHTML` here? If you expect it to be sanitized elsewhere, please document that.